tools/validators/instance_validator/tests/telemetry_validator_test.py

```diff
@@ -194,7 +194,7 @@ class TelemetryValidatorTest(absltest.TestCase):
 
   def testTelemetryValidatorGetErrorsReturnsAllErrors(self):
     validator = telemetry_validator.TelemetryValidator(GOOD_ENTITIES_1, 1,
-                                                       _NullCallback)
+                                                       False, _NullCallback)
```

Comment:
can you please set `isUdmi=False` here and in the tests below

Reply:
I think I've made the change requested, but please take a look and let me know because I wasn't sure if I understood it properly.